### PR TITLE
Display long form content on discover tab

### PR DESCRIPTION
**Pull request by boreq**

### CODE REVIEW

**Comment by boreq** [comment]
Screenshot 2023-09-21 at 18 18 14

File: Nos/Models/Event+CoreDataClass.swift [hunk 1]
```diff
@@ -77,6 +77,8 @@ public class Event: NosManagedObject {
     
     static var replyNoteReferences = "kind = 1 AND ANY eventReferences.referencedEvent.identifier == %@ " +
         "AND author.muted = false"
+    public static var discoverKinds = [EventKind.text, EventKind.longFormContent]
```
Comment (boreq) [inline Nos/Models/Event+CoreDataClass.swift:80]: I think it makes sense to define this somewhere but unfortunately I don't know where. Suggestions are very much welcome.

Comment (mplorentz) [reply Nos/Models/Event+CoreDataClass.swift:80]: this works! I might have put it on `EventKind` but this makes just as much sense I think.

**Review by mplorentz — APPROVED** [review]
Perfection!

**Review by martindsq — APPROVED** [review]

**Pull request merged**